### PR TITLE
Changing code format from ES5 to ES6

### PR DESCRIPTION
**Pull request by Duncanwar**
#### What does this PR do?
This PR changes some file code from ES5 to ES6.
#### Description of Task to be completed?
- Used ES6 import feature for importing packages
#### How should this be manually tested?
- run :
-  `git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend.git`
- `cd teck-strickers-bn-backend`
- `git checkout ch-change-code-format`
- `npm install`
Open `index.js` file in src directory you will see ES6 import feature
#### Any background context you want to provide?
- Removed all mongoose related stuff
#### Screenshots (if appropriate)
Screenshot from 2020-11-18 14-51-24

#### Questions:
Can I structure our project and refactor the code we have now?

### CODE REVIEW

File: src/routes/api/index.js [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
 
-router.use((err, req, res, next) => {
-  if (err.name === "ValidationError") {
-    return res.status(422).json({
-      errors: Object.keys(err.errors).reduce((errors, key) => {
-        errors[key] = err.errors[key].message;
-        return errors;
-      }, {}),
-    });
-  }
```
Comment (MChristian00) [inline src/routes/api/index.js:13]: Did we have to remove all these?

File: src/models/User.js [hunk 2]
```diff
@@ -1,55 +0,0 @@
...
-    .pbkdf2Sync(password, this.salt, 10000, 512, "sha512")
-    .toString("hex");
-};
-
-UserSchema.methods.toAuthJSON = () => ({
-  username: this.username,
-  email: this.email,
-});
-
-mongoose.model("User", UserSchema);
```
Comment (MChristian00) [inline src/models/User.js]: I thought some of these were to be used as a reference. Won't it be an issue if we remove them?

File: src/index.js [hunk 3]
```diff
@@ -38,20 +32,7 @@ app.use(
...
-if (!isProduction) {
-  app.use(errorhandler());
-}
-
-if (isProduction) {
-  mongoose.connect(process.env.MONGODB_URI);
-} else {
-  mongoose.connect("mongodb://localhost/conduit");
-  mongoose.set("debug", true);
-}
```
Comment (MChristian00) [inline src/index.js]: I think we can just erase lines containing mongoose like line 46,48,49

**Review by MChristian00 — CHANGES_REQUESTED** [review]
GREAT WORK,
Just left my reviews on some files.

File: src/models/User.js (hunk 2 above)
Comment (Duncanwar) [reply src/models/User.js]: UserSchema has reference to mongoose model. That is the reason I deleted all of them

Comment (Duncanwar) [reply src/models/User.js]: No. All these are related to mongoose which is a no-SQL database and we are going to use Postgres

**Comment by bigelio** [comment]
Great work @Duncanwar and remember to include story id in title of your PR.
Make sure that  the feature can be tested manually because I'm trying to test it manually and get this error
Screen Shot 2020-11-18 at 19 40 49

**Comment by Duncanwar** [comment]
For me it is working

**Comment by bigelio** [comment]
Can you do rebase again please?

File: src/index.js [hunk 4]
```diff
@@ -40,26 +37,23 @@ app.use(
...
-//CONNECTING APP TO POSTGRESS
+// CONNECTING APP TO POSTGRESS
 
-  const sequelize = new Sequelize(development.url, {
-    host: host,
-    dialect: 'postgres' 
-  });
+const sequelize = new Sequelize(development.url, {
+  dialect: "postgres"
+});
```
Comment (MChristian00) [inline src/index.js:44]: You've removed the `host`.

**Review by MChristian00 — COMMENTED** [review]
Looks well.
Except for the following issue.

**Comment by Duncanwar** [comment]
I am using actual URL and there is no need of the host

**Pull request merged**